app/controllers/feedback_controller.rb

```diff
@@ -0,0 +1,21 @@
+class FeedbackController < ApplicationController
+  def new
+    @feedback = Feedback.new(url: params[:url])
```

Comment:
I'll have to check this to make sure it works with multiple form submissions. Should be ok. Good suggestion.